tex/gregoriotex.lua

```diff
@@ -1237,119 +1203,107 @@ local function locate_file(filename)
   return result
 end
 
-local function include_score(input_file, force_gabccompile, allow_deprecated)
-  if string.match(input_file, "[#%%]") then
+local function include_score(gabc_file, force_gabccompile, allow_deprecated)
+  local sep = ""
+  local onwindows = (os.type == "windows" or
+                     string.find(os.getenv("PATH"),";",1,true))
+  if onwindows then
+    sep = "\\"
```

Comment:
Something appears to be going wrong here.  Maybe we should be getting rid of the whole sep distinction here and rely on `lfs.normalize` to make sure our path separators are correct.